include/samurai/mr/mesh.hpp

```diff
@@ -339,11 +341,14 @@ namespace samurai
                 this->cells()[mesh_id_t::all_cells][level + 1] = lcl;
                 this->cells()[mesh_id_t::proj_cells][level]    = lcl_proj;
             }
-            this->update_mesh_neighbour();
+            // this->update_mesh_neighbour();
+            this->update_neighbour_subdomain();
+            this->update_neighbour_all_cells();
         }
         else
         {
             this->cells()[mesh_id_t::all_cells] = {cell_list, false};
+            // TODO : what to send here ?
```

Comment:
In this case, you have `min_level = max_level` which means that you don't have any refinement. Then the `cell_and_ghost` mesh is the same as the `reference`. So, you probably need to call `update_neighbour_cells_and_ghosts()` and `update_neighbour_reference()`.